src/wwwroot/controllers/MainCtrl.js

```diff
@@ -33,8 +33,8 @@
     var loaderFreeTrial = function () {
       if (auth.getUserName() != null) {
         auth.getLimitsByAccount().then(function(limits) {
-          if (limits.freeTrialEndDate) {
-            UpdateTrialHeader(moment(limits.freeTrialEndDate).toDate());
+          if (limits.data.freeTrialEndDate) {
+            UpdateTrialHeader(moment(limits.data.freeTrialEndDate).toDate());
```

Comment:
👍 

By the way, could you rename _freeTrialEndDate_ as _endDate_ or _accountEndDate_?

Reply:
as `endDate` I'll rename